CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This release is rolling out to all regions - estimated time for completed roll o
   * Fixed bug with auto-scaling from zero with pods that utilize an `agentpool=` label selector.
   * Fixed bug for IPv6-enabled clusters using OpenVPN and BYO VNET that checked the incorrect IPv6 CIDR.
 * Behavioral changes
+  * An empty or partial PUT following a control plane upgrade, without any agent pool specified will no longer upgrade all the agent pools. 
```

Comment:
"An empty or partial PUT on managed cluster without any agent pool specified will no longer upgrade all the agent pools. This was causing too many unintentional upgrades. 
Control plane will be upgaded if specified and no agentpools are upgraded. But you will get an error if you specify only some agnetpools and not others
If you wish to upgrade agentpools please specify ALL agentpools orchestratorVersion. Ideally a full GET, mutate and put is recommended here."